ansible/roles/providers/tasks/googlecompute.yml

```diff
@@ -40,3 +40,51 @@
     path: "/bin/{{ item.path | basename }}"
     state: link
   with_items: "{{ find.files }}"
+
+- name: Install cloud-init packages
+  apt:
+    name: "{{ packages }}"
+    state: present
+    force_apt_get: true
+  vars:
+    packages:
+      - cloud-init
+      - cloud-guest-utils
+      - cloud-initramfs-copymods
+      - cloud-initramfs-dyn-netconf
```

Comment:
cloud-init is a requirement for CAPA. Is there any way we can consolidate this stuff to avoid drift between cloud providers

Reply:
It seems like image-builder still expected that cloud-init be installed, specially for ubuntu distros. There are checks for these also so I assume that the assumption is that this still exists.

Reply:
I see. The point i was making is that `providers/tasks/vsphere.yaml` does something very similar in installing packages and was wondering if we could consolidate those installs in one file.